packages/rum-recorder/src/domain/rrweb-snapshot/types.ts

```diff
@@ -0,0 +1,97 @@
+export enum NodeType {
+  Document,
+  DocumentType,
+  Element,
+  Text,
+  CDATA,
+  Comment,
+}
+
+export type DocumentNode = {
+  type: NodeType.Document
+  childNodes: SerializedNodeWithId[]
+}
+
+export type DocumentTypeNode = {
+  type: NodeType.DocumentType
+  name: string
+  publicId: string
+  systemId: string
+}
+
+export type Attributes = {
+  [key: string]: string | number | boolean
+}
+export type ElementNode = {
+  type: NodeType.Element
+  tagName: string
+  attributes: Attributes
+  childNodes: SerializedNodeWithId[]
+  isSVG?: true
+  needBlock?: boolean
+}
+
+export type TextNode = {
+  type: NodeType.Text
+  textContent: string
+  isStyle?: true
+}
+
+export type CDataNode = {
+  type: NodeType.CDATA
+  textContent: ''
+}
+
+export type CommentNode = {
+  type: NodeType.Comment
+  textContent: string
+}
+
+export type SerializedNode = DocumentNode | DocumentTypeNode | ElementNode | TextNode | CDataNode | CommentNode
+
+export type SerializedNodeWithId = SerializedNode & { id: number }
+
+export type tagMap = {
```

Comment:
This type doesn't seem to be used. Could you remove it? else, change it to `TagMap`

Reply:
good catch! latest commit removes it